zipkin-autoconfigure/collector-kafka10/README.md

```diff
@@ -10,48 +10,48 @@ variable or `zipkin.collector.kafka.bootstrap-servers` property.
 JRE 8 is required to run Zipkin server. Note: The Kafka 0.10+ collector and this auto-configure
 module are compatible with Java 7 and later when used independent of Zipkin server.
 
-Fetch the latest released 
+Fetch the latest released
 [executable jar for Zipkin server](https://search.maven.org/remote_content?g=io.zipkin.java&a=zipkin-server&v=LATEST&c=exec)
```

Comment:
Unfortunately this link does not work due to the ongoing Maven central certificate problem. I'm not sure what's best for us to do about it, though.

Reply:
indeed see https://github.com/openzipkin/zipkin/issues/1804 and specifically https://issues.sonatype.org/browse/MVNCENTRAL-2870 which covers the root problem (those watching from home)

Reply:
https://github.com/openzipkin/zipkin/issues/1836